README.md

```diff
@@ -88,14 +88,14 @@ Note: to proceed with this you need to have write authorization to npm.
 
 1. Create a new PR with updated `CHANGELOG.md` and updated version in `package.json` (i.e. `1.2.3-alpha.1`), see an example [here](https://github.com/Automattic/gridicons/pull/247).
 2. Pre-publish that PR branch on npm with `npm publish --tag next` ([more info](https://docs.npmjs.com/cli/dist-tag)).
-3. Create a new PR in a repository using Gridicons (i.e. [Calypso](https://github.com/Automattic/wp-calypso)), see an example [here](https://github.com/Automattic/wp-calypso/pull/17601).
+3. Create a new update PR in a repository that makes use of Gridicons (i.e. [Calypso](https://github.com/Automattic/wp-calypso)) pointing `package.json` to the alpha version, see an example [here](https://github.com/Automattic/wp-calypso/pull/17601).
 4. Test if the new icons show up, and there are no regressions in the previous icons.
-5. If changes look good, remove postfix in the version (i.e. `1.2.3`).
+5. If changes look good, remove the alpha postfix in the version (i.e. `1.2.3-alpha.1` to `1.2.3`) on both repositories PRs.
 6. Merge the Gridicons PR.
 7. Tag the release on GitHub: `git tag -a v1.2.3 -m "Release v1.2.3"` (and push `git push origin v1.2.3`).
 8. Check if it shows up in the [Releases list](https://github.com/Automattic/gridicons/releases).
 8. Publish to MASTER using the latest tag `npm publish`.
-9. Merge the test repository PR.
+9. Merge the update PR in the other repository.
```

Comment:
Updated?

Reply:
It's meant to match point 3: "Create a new update PR in a repository...", but probably there's a better way to use the same wording on 3 and 9 so the reader know it's the same PR?